docs/data/ingest-library/developer_guide/ledgerbackends/captivecore.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Captive Core
+sidebar_position: 0
+---
+
+Captive Core is used to either ingest ledgers in real-time or replay historical ledgers without any data processing. It connects to the Stellar network and replays historical ledgers or streams new ones.
```

Comment:
```suggestion
Captive Core invokes the `stellar-core` binary as a subprocess to stream ledgers from the Stellar network. It can be used to stream a ledger range from the past or to stream new ledgers whenever they are confirmed by the network.
```